wallet/src/Pos/Wallet/Web/Methods/Logic.hs

```diff
@@ -127,8 +133,9 @@ getAccountsIncludeUnready includeUnready mCAddr = do
     accIds <- maybe getAccountIds getWalletAccountIds mCAddr
     let groupedAccIds = fmap reverse $ HM.fromListWith mappend $
                         accIds <&> \acc -> (aiWId acc, [acc])
+    balAndUtxo <- WS.getWalletBalancesAndUtxo
     concatForM (HM.toList groupedAccIds) $ \(wid, walAccIds) ->
-         fixCachedAccModifierFor wid $ forM walAccIds . getAccount
+         fixCachedAccModifierFor wid $ forM walAccIds . getAccountMod . (balAndUtxo,)
```

Comment:
We could simplify this, it's getting unnecessarily complicated. The `fixCachedAccModifierFor` is really just a `getCachedAccModifier :: CId Wal -> m CachedCAccModifier`. So this could be just:

```
concatForM (HM.toList groupedAccIds) $ \(wid, walAccIds) -> do
    cachedAccModifier <- getCachedAccModifier wid
    mapM (getAccount balAndUtxo cachedAccModifier) walAccIds
```